### PR TITLE
Adding info in Password Options doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request describes common feature information for better understanding, info about 'Enable Autocomplete on login/forgot password forms' field was missing in the doc

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Password Options

## Affected documentation pages

https://docs.magento.com/user-guide/customers/password-options.html

## Links to Magento source code or PRs

https://docs.magento.com/user-guide/marketing/event-restrict-access.html

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

File: src/customers/password-options.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ The customer password options determine the level of security that is used for p
 
    - For **Recovery Link Expiration Period (hours)**, enter the number of hours before the password recovery link expires.
 
+   - For **Enable Autocomplete on login/forgot password forms**, set to `Yes`, if you want the fields in the customer login and forgot password forms to be filled automatically from previous entries
```
Comment (leslietilling) [inline src/customers/password-options.md:46]:
```suggestion
   - If you want the fields in the customer login and forgot password forms to be filled automatically from previous entries, set **Enable Autocomplete on login/forgot password forms** to `Yes`.
```

Comment (dnahirna) [reply src/customers/password-options.md:46]: Hi, @leslietilling. Do you want me replace this text?

Comment (dnahirna) [reply src/customers/password-options.md]: done

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**